tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -255,10 +269,34 @@ var actionGetCmd = &cobra.Command{
     if flags.common.summary {
       printSummary(action)
     } else {
-      fmt.Fprintf(color.Output,
-        wski18n.T("{{.ok}} got action {{.name}}\n",
-          map[string]interface{}{"ok": color.GreenString("ok:"), "name": boldString(qName.entityName)}))
-      printJSON(action)
+
+      if len(field) > 0 {
+        fmt.Fprintf(color.Output, wski18n.T("{{.ok}} got action {{.name}}, projecting {{.field}}\n",
+          map[string]interface{}{"ok": color.GreenString("ok:"), "name": boldString(qName.entityName), "field": field}))
+
+        if field == "namespace" {
```

Comment:
Comment applies to similar action, activation, package, rule, and trigger code

It may be possible to create a common print function to handle displaying the filtered field value.  In that function, use the reflect package to reference the target structure field based on the field filter argument.  Maybe something like:   printFieldJSON(entity interface{}, field string)